x-pack/plugins/security_solution/public/common/components/markdown/index.test.tsx

```diff
@@ -1,176 +0,0 @@
-    const wrapper = mount(<Markdown raw={raw} />);
-
-    expect(wrapper.find('[data-test-subj="markdown-root"]').first().text()).toEqual(raw);
-  });
-
-  test('it applies the EUI text style to all markdown content', () => {
-    const wrapper = mount(<Markdown raw={'#markdown'} />);
-
-    expect(
-      wrapper.find('[data-test-subj="markdown-root"]').first().childAt(0).hasClass('euiText')
-    ).toBe(true);
-  });
-
-  describe('markdown tables', () => {
-    const headerColumns = ['we', 'support', 'markdown', 'tables'];
-    const header = `| ${headerColumns[0]} | ${headerColumns[1]} | ${headerColumns[2]} | ${headerColumns[3]} |`;
-
-    const rawTable = `${header}\n|---------|---------|------------|--------|\n| because | tables  | are        | pretty |\n| useful  | for     | formatting | data   |`;
-
-    test('it applies EUI table styling to tables', () => {
-      const wrapper = mount(<Markdown raw={rawTable} />);
-
-      expect(wrapper.find('table').first().childAt(0).hasClass('euiTable')).toBe(true);
-    });
-
-    headerColumns.forEach((headerText) => {
-      test(`it renders the "${headerText}" table header`, () => {
-        const wrapper = mount(<Markdown raw={rawTable} />);
-
-        expect(wrapper.find('[data-test-subj="markdown-table-header"]').first().text()).toContain(
-          headerText
-        );
-      });
-    });
-
-    test('it applies EUI table styling to table rows', () => {
-      const wrapper = mount(<Markdown raw={rawTable} />);
-
-      expect(
-        wrapper
-          .find('[data-test-subj="markdown-table-row"]')
-          .first()
-          .childAt(0)
-          .hasClass('euiTableRow')
-      ).toBe(true);
-    });
-
-    test('it applies EUI table styling to table cells', () => {
-      const wrapper = mount(<Markdown raw={rawTable} />);
-
-      expect(
-        wrapper
-          .find('[data-test-subj="markdown-table-cell"]')
-          .first()
-          .childAt(0)
-          .hasClass('euiTableRowCell')
-      ).toBe(true);
-    });
-
-    test('it renders the expected table content', () => {
-      const wrapper = shallow(<Markdown raw={rawTable} />);
-
-      expect(wrapper).toMatchSnapshot();
-    });
-  });
-
-  describe('markdown links', () => {
-    const markdownWithLink = 'A link to an external site [External Site](https://google.com)';
-
-    test('it renders the expected link text', () => {
-      const wrapper = mount(<Markdown raw={markdownWithLink} />);
-
-      expect(wrapper.find('[data-test-subj="markdown-link"]').first().text()).toEqual(
-        'External Site'
-      );
-    });
-
-    test('it renders the expected href', () => {
-      const wrapper = mount(<Markdown raw={markdownWithLink} />);
-
-      expect(wrapper.find('[data-test-subj="markdown-link"]').first().getDOMNode()).toHaveProperty(
-        'href',
-        'https://google.com/'
-      );
-    });
-
-    test('it does NOT render the href if links are disabled', () => {
```

Comment:
The intent of the behavior being asserted in this test is to disable links when the `disableLinks={true}` prop is passed. This is used, for example, to disable links in case descriptions on the Overview page. Would you be willing to restore this behavior and add an equivalent test?